SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/TestMovementCharacter.cpp

```diff
@@ -41,8 +42,24 @@ ATestMovementCharacter::ATestMovementCharacter()
 	CameraComponent->SetRelativeRotation(CameraRotation);
 #endif
 	CameraComponent->SetupAttachment(GetCapsuleComponent());
+
+	// Increase MaxSimulationIterations to 14 so that we do not get warnings inside UCharacterMovementComponent::GetSimulationTimeStep
```

Comment:
ah, funny, I have seen this warning in EP, maybe we should take a look at some point